info.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/golang/gddo/gosrc"
+)
+
+type defInfo struct {
+	Package  string
+	Path     string
+	Repo     string
+	Unit     string
+	UnitType string
+	Kind     string
+}
+
+func outputData(data ...interface{}) {
+	output := fmt.Sprintln(data...)
+	if !*useJSON {
+		fmt.Print(output)
+		return
+	}
+	printStructured(output)
+}
+
+func printStructured(output string) {
+	// TODO main package files
+	datas := strings.Split(strings.Trim(output, "\n"), " ")
```

Comment:
i dont think it's good that you first call `fmt.Sprintln` on this to combine the thing into a single string, then unparse it here...just pass through the `[]interface{}`? (which is always strings, right? so just `[]string`?)